spec/lib/addressfinder/cleanse_spec.rb

```diff
@@ -98,7 +98,7 @@
     context 'with a question mark value' do
       let(:q){ '?' }
 
-      it { expect(encoded_params).to eq('q=%3F&format=json&key=XXX&secret=YYY') }
+      it { expect(encoded_params).to eq('q=?&format=json&key=XXX&secret=YYY') }
```

Comment:
that question mark appears incorrect. I would have thought it should have been encoded as the %3F